### PR TITLE
Add github actions for package and release on pypi

### PR DESCRIPTION
**Pull request by guillaumebert**
## What?
New Continuous integration environment in Github.

## Why?
Because our CI travis.org must be migrated and jobs are queued for a long time.

## How?
2 new pipelines :
 - "Python package" to build, test and apply sonar analysis. Triggered at each push on a branch and each pull request to master.
 - "Python release" to package and upload on pypi. Triggered at each push a new tag.

## Testing
This pull request should be built.

2 versions "1.1.5.dev1" and  "1.1.5.dev2" has been published on pypi to test the pipeline release.

## Additional Notes
⚠️ ⚠️ **Before merging we should disable travis automatic builds when we push on github.**

### CODE REVIEW

**Comment by YMasliah-neotys** [comment]
I have no issues with it (if you remove travis contineous integration for it to be mergeable xD)

I will not valide it anyway because i dont know how python works. i let other valide this PR.

BTW SonarCloud recommended me to cache SonarCloud packages because they will be used in all PR.
I dont know if the plugin used for SonarCloud for python pipelines have this cache done in it.

The code i needed to add in my git actions: (found somewhere on SonarCloud doc)
```
      - name: Cache SonarCloud packages
        uses: actions/cache@v2
        with:
          path: ~/.sonar/cache
          key: ${{ runner.os }}-sonar
          restore-keys: ${{ runner.os }}-sonar 
```

**Comment by paulsbruce** [comment]
When should we disable travis builds? Once we do that, I think we can give this an official go ahead.

**Comment by guillaumebert** [comment]
We can disable travis builds as soon as we are ready to merge (when this PR is approved).
About Sonar, I didn't receive any recommandation to add cache.

**Review by neotys-rd — APPROVED** [review]

**Pull request merged**
